fabric-lifecycle-events-v1/src/client/java/net/fabricmc/fabric/api/client/event/lifecycle/v1/ClientWorldEvents.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.client.event.lifecycle.v1;
+
+import org.jetbrains.annotations.Nullable;
+
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.world.ClientWorld;
+
+import net.fabricmc.fabric.api.event.Event;
+import net.fabricmc.fabric.api.event.EventFactory;
+
+public final class ClientWorldEvents {
+	private ClientWorldEvents() {
+	}
+
+	/**
+	 * An event which is called after the client world has been changed.
+	 */
+	public static final Event<AfterClientWorldChange> AFTER_CLIENT_WORLD_CHANGE = EventFactory.createArrayBacked(AfterClientWorldChange.class, callbacks -> (client, world) -> {
```

Comment:
I believe this event should be called `LOAD` due to these two reasons:
1. To be consistent with the `ServerWorldEvents` class.
2. This event is also called on the initial world load, and it doesn't make sense to be called `AFTER_CLIENT_WORLD_CHANGE`.

Also, I assume this event only fires once on world change, not once `null` then once with the new world?

Reply:
That makes sense, I have completed the modifications.

Reply:
I dont think load is correct at all, when joining a server, or switching a world you arent loading (from the disk) the world at all.

Reply:
Should it be reverted back to "AFTER_CLIENT_WORLD_CHANGE", or a shorter name "CHANGE" be more appropriate?

Reply:
I have reverted its name now.